app.js

```diff
@@ -295,15 +327,15 @@ var eventsModule = function () {
             var bulkQuery = {};
             templateUsed.frames.forEach(EF => {
                 var attributeURL;
-                attributeURL = encodeURI(webAPIServerURL + "/streamsets/" + EF.id + "/value?nameFilter=" + attributeName + "&selectedFields=Items.Value.Value");
+                attributeURL = encodeURI(webAPIServerURL + "/streamsets/" + EF.id + "/value?selectedFields=Items.Name;Items.Value.Value");
```

Comment:
I think it makes sense to get all of the attributes now that we have a use!!  Not much of a performance cost due to how the web api does the caching for EF attributes.